nautobot/ipam/models.py

```diff
@@ -1065,13 +1065,17 @@ def _deconstruct_address(self, address):
     natural_key_field_names = ["parent__namespace", "host"]
 
     def _get_closest_parent(self):
+        if closes_parent := getattr(self, "_closest_parent", None):
+            return closes_parent
```

Comment:
```suggestion
        if getattr(self, "_closest_parent", None):
            return self._closest_parent
```

The `:=` operator isn't really necessary here and I'd rather omit it when not needed because it's not that common so not everybody understands how it works.

Reply:
This class has not attribute `_closest_parent`, doing that would throw an error, especially when its not yet set

Reply:
If it's not set yet you will get `None` and the block will be skipped?

Reply:
This isn't on IPAddressForm though, it's on the IPAddress model.

Reply:
Gary's suggested change here looks correct to me. Can you help me understand why it's incorrect?

Reply:
calling `self._closest_parent` would throw this error `AttributeError: 'IPAddress'  has no attribute '_closest_parent'` not `None`, since `IPAddress` has no attribute called `_closest_parent `.

But this has been resolved